MyProject/src/main/java/com/netcracker/model/Devices.java

```diff
@@ -0,0 +1,35 @@
+package com.netcracker.model;
+
+import com.arangodb.springframework.annotation.Document;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import org.springframework.data.annotation.Id;
+
+@FieldDefaults(level = AccessLevel.PRIVATE)
```

Comment:
I would recommend to specify level explicitly for each filed and not use this annotation. The code will be more readable.